### PR TITLE
feat: include 'reaction and comment count' in GET /posts

### PR DESCRIPTION
**Pull request by RickaPrincy**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -48,13 +49,14 @@ model Comment {
 }
 
 model User {
-  id       String    @id @default(uuid())
-  email    String    @unique
-  password String    @db.VarChar(100)
-  username String    @db.VarChar(50)
-  joinedAt DateTime  @default(now())
+  id       String     @id @default(uuid())
```
Comment (YumeT023) [inline prisma/schema.prisma:52]: it is not yet formatted

`npx prisma format` to format it correctly

**Review by YumeT023 — CHANGES_REQUESTED** [review]

**Comment by RickaPrincy** [comment]
Done

File: prisma/schema.prisma (hunk 1 above)
Comment (RickaPrincy) [reply prisma/schema.prisma:52]: Done

Comment (RickaPrincy) [reply prisma/schema.prisma]: Done

**Pull request merged**